.github/workflows/smoke-test-release.yml

```diff
@@ -93,10 +91,19 @@ jobs:
         run: |
           ASSET_ID=$(jq ".release.assets[0].id" $GITHUB_EVENT_PATH)
 
-          curl https://api.github.com/repos/woocommerce/woocommerce/releases/assets/${ASSET_ID} -LJOH 'Accept: application/octet-stream'
+          curl https://api.github.com/repos/woocommerce/woocommerce/releases/assets/52302540 -LJOH 'Accept: application/octet-stream'
```

Comment:
This change will also need to be reverted so that subsequent releases will be tested against the appropriate asset.